src/utils/date-time-formatter.js

```diff
@@ -385,18 +385,22 @@ DateTimeFormatter.getTokenDefinitions = function () {
             index: 0,
             extract () { return '(\\d{2})'; },
             parser (val) {
+                let result;
                 if (val) {
                     const l = val.length;
                     val = val.substring(l - 2, l);
                 }
                 let parsedVal = DateTimeFormatter.defaultNumberParser()(val);
                 let presentYear = Math.trunc(((new Date()).getFullYear()) / 100);
                 if (parsedVal instanceof Number) {
-                    parsedVal = (presentYear * 100) + parsedVal;
+                    result = `${(presentYear * 100) + parsedVal}`;
                 } else {
-                    parsedVal = `${presentYear}${parsedVal}`;
+                    result = `${presentYear}${parsedVal}`;
                 }
-                return parsedVal;
+                if (new Date(result).getFullYear() > new Date().getFullYear()) {
```

Comment:
Store current date and result date full year in a variable

Reply:
Why is that ?

Reply:
done.. Instead of the used `convertToNativeDate`